packages/synapse-react-client/src/components/EvaluationFinder/EvaluationFinder.tsx

```diff
@@ -35,7 +35,7 @@ export default function EvaluationFinder(props: EvaluationFinderProps) {
       accessType,
       activeOnly,
     },
-    { keepPreviousData: true, useErrorBoundary: true },
+    { placeholderData: previousData => previousData, throwOnError: true },
```

Comment:
`keepPreviousData`  was removed. To achieve the same behavior, we can pass a function to `placeholderData`  and return the `previousData` without making any changes.